src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelBindingContext.cs

```diff
@@ -260,28 +257,27 @@ public ModelStateDictionary ModelState
         /// Passed into the model binding system. Should not be <c>true</c> when <see cref="IsTopLevelObject"/> is
         /// <c>false</c>.
         /// </remarks>
-        public bool FallbackToEmptyPrefix { get; set; }
+        public bool FallbackToEmptyPrefix { get { return _state.FallbackToEmptyPrefix; } set { _state.FallbackToEmptyPrefix = value; } }
```

Comment:
Long enough to start wrapping this. Same for a few other properties in this class.